chartiq/development/js/chartiq.js

```diff
@@ -14628,7 +14628,7 @@ CIQ.ChartEngine.prototype.adjustBackingStore = function (canvas, context) {
 };
 
 CIQ.ChartEngine.prototype.reconstituteBackingStore = function () {
-	if (!this.useBackingStore) return;
+	if (!this.useBackingStore || !this.backing) return;
```

Comment:
@kseniya-deriv the `chartiq` library comes from a 3rd party library so we cannot change its code directly. we modify the chartiq code using code injection or monkey patching, you can find injections in `src/chartiq_injections/backingStore.js`